<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingPage</class>
 <widget class="QWidget" name="SettingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 11px &quot;Verdana&quot;;
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="baseHeadingLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>710</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18px &quot;Verdana&quot;;
font-weight: bold;
color: rgb(140, 192, 65);</string>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="generalBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>711</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
</string>
   </property>
   <property name="title">
    <string>General Settings</string>
   </property>
   <widget class="QCheckBox" name="custPrefixCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: normal;</string>
    </property>
    <property name="text">
     <string>Use and enforce customer prefix (Decimal)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="custPrefixTxt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: normal;</string>
    </property>
    <property name="inputMask">
     <string>00000; </string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="loggingBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>422</y>
     <width>711</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
</string>
   </property>
   <property name="title">
    <string>Logging Settings</string>
   </property>
   <widget class="QCheckBox" name="logOutputCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: normal;</string>
    </property>
    <property name="text">
     <string>Log configuration output</string>
    </property>
   </widget>
   <widget class="QLabel" name="logFileLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:normal;</string>
    </property>
    <property name="text">
     <string>Log Output File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="logFileTxt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: normal;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="browseBtn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>48</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="actionsBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>512</y>
     <width>711</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;</string>
   </property>
   <property name="title">
    <string>Actions</string>
   </property>
   <widget class="QPushButton" name="saveBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>85</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="actionsDescLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>691</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: normal;
color: rgb(118, 116, 108)
</string>
    </property>
    <property name="text">
     <string>Press Save to save your settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restoreBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>45</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Restore Defaults</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="outputBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>711</width>
     <height>180</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
</string>
   </property>
   <property name="title">
    <string>Output Settings</string>
   </property>
   <widget class="QGroupBox" name="outFormatBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>691</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;
</string>
    </property>
    <property name="title">
     <string>Output Format</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="tabFirstBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>68</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Tab</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="appendTab1Btn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>25</y>
       <width>68</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Tab</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="appendTab2Btn">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>25</y>
       <width>68</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Tab</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="pubIdLbl">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>25</y>
       <width>52</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight:normal;</string>
     </property>
     <property name="text">
      <string>Public ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="otpLbl">
     <property name="geometry">
      <rect>
       <x>258</x>
       <y>25</y>
       <width>32</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight:normal;</string>
     </property>
     <property name="text">
      <string>OTP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="appendCRBtn">
     <property name="geometry">
      <rect>
       <x>398</x>
       <y>25</y>
       <width>68</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="outFormatHelpBtn">
     <property name="geometry">
      <rect>
       <x>665</x>
       <y>0</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/res/images/question.png);
color: lightgray;
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="outSpeedBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>691</width>
      <height>77</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;
</string>
    </property>
    <property name="title">
     <string>Output Speed Throttling</string>
    </property>
    <widget class="QPushButton" name="outSpeedHelpBtn">
     <property name="geometry">
      <rect>
       <x>665</x>
       <y>0</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/res/images/question.png);
color: lightgray;
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="outCharRateCombo">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>24</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: normal;</string>
     </property>
     <item>
      <property name="text">
       <string>Standard</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Slow down by 20 ms</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Slow down by 40 ms</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Slow down by 60 ms</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="outCharRateLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight:normal;</string>
     </property>
     <property name="text">
      <string>Output Character Rate</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="appendDelay1Check">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>281</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: normal;</string>
     </property>
     <property name="text">
      <string>Add a short delay before sending OTP part</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="appendDelay2Check">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>271</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: normal;</string>
     </property>
     <property name="text">
      <string>Add a short delay after sending OTP part</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="srVisibilityBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>350</width>
     <height>102</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
</string>
   </property>
   <property name="title">
    <string>Serial # Visibility Settings (YubiKey 2.2)</string>
   </property>
   <widget class="QCheckBox" name="srBtnVisibleCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: normal;</string>
    </property>
    <property name="text">
     <string>Button at startup</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="srUsbVisibleCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: normal;</string>
    </property>
    <property name="text">
     <string>USB descriptor</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="srApiVisibleCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: normal;</string>
    </property>
    <property name="text">
     <string>API call</string>
    </property>
   </widget>
   <widget class="QPushButton" name="srVisibilityHelpBtn">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>4</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/res/images/question.png);
color: lightgray;
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="staticBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>350</width>
     <height>102</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
</string>
   </property>
   <property name="title">
    <string>Static Password Setttings</string>
   </property>
   <widget class="QCheckBox" name="manUpdateCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>331</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: normal;</string>
    </property>
    <property name="text">
     <string>Enable manual update using the button (YubiKey 2)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="updateBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>350</width>
     <height>102</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
</string>
   </property>
   <property name="title">
    <string>Update Settings (YubiKey 2.3)</string>
   </property>
   <widget class="QCheckBox" name="updateCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>331</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: normal;</string>
    </property>
    <property name="text">
     <string>Enable updating of YubiKey configuration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateHelpBtn">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>5</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/res/images/question.png);
color: lightgray;
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>custPrefixCheck</tabstop>
  <tabstop>custPrefixTxt</tabstop>
  <tabstop>tabFirstBtn</tabstop>
  <tabstop>appendTab1Btn</tabstop>
  <tabstop>appendTab2Btn</tabstop>
  <tabstop>appendCRBtn</tabstop>
  <tabstop>outCharRateCombo</tabstop>
  <tabstop>appendDelay1Check</tabstop>
  <tabstop>appendDelay2Check</tabstop>
  <tabstop>srBtnVisibleCheck</tabstop>
  <tabstop>srUsbVisibleCheck</tabstop>
  <tabstop>srApiVisibleCheck</tabstop>
  <tabstop>manUpdateCheck</tabstop>
  <tabstop>logOutputCheck</tabstop>
  <tabstop>logFileTxt</tabstop>
  <tabstop>browseBtn</tabstop>
  <tabstop>saveBtn</tabstop>
  <tabstop>restoreBtn</tabstop>
  <tabstop>outFormatHelpBtn</tabstop>
  <tabstop>outSpeedHelpBtn</tabstop>
  <tabstop>srVisibilityHelpBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
